backend/api/v1/endpoints/chat.py

```
@@ -2,7 +2,7 @@
 import uuid
 from urllib.parse import urlparse
 from fastapi import APIRouter, File, HTTPException, UploadFile, WebSocket, WebSocketDisconnect, Depends
-from fastapi.responses import JSONResponse
+from fastapi.responses import JSONResponse, FileResponse
```

Ruff (0.8.2)

`fastapi.responses.FileResponse` imported but unused

Remove unused import: `fastapi.responses.FileResponse`

(F401)